### ISSUE TITLE
System.PlatformNotSupportedException when CTRL+L

### ISSUE THREAD

**Issue by sorge13248** [PowerShell/PSReadLine#922]
Stack trace
----------------
Last 2 Keys:
```
 Ctrl+l Ctrl+l
```

Exception:
```
System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.ConsolePal.SetWindowPosition(Int32 left, Int32 top)
   at System.Console.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ClearScreen(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

Environment data
----------------
PS version:  6.2.0
PSReadline version: 2.0.0-beta3
os: Linux shi-tri-lin 4.15.0-48-generic #51-Ubuntu SMP Wed Apr 3 08:28:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
PS file version: 

Steps to reproduce or exception report
--------------------------------------
1. Launch PoweShell `pwsh`
2. Press CTRL+L two times
3. Exception happens

**Comment by SteveL-MSFT** [comment]
This doesn't repro on 2.0.0-beta.4 on Ubuntu 18.04

**Comment by lzybkr** [comment]
@SteveL-MSFT - This has been reported multiple times and likely is still an issue as nothing has been done to avoid the exception. But you might tag it as a dupe when you find the similar issues.

**Comment by daxian-dbw** [comment]
close as a duplicate of #991

**Comment by daxian-dbw** [comment]
Keep this issue open, as #991 is originally about a `ArgumentOutOfRangeException`.

**Comment by daxian-dbw** [comment]
I took a closer look at this issue today and I think the issue had already been fixed by https://github.com/PowerShell/PSReadLine/pull/790, which [changed the implementation of `ClearScreen`](https://github.com/PowerShell/PSReadLine/commit/bc40ae6f6b44e2eb4bd8b57810079bfd89bffa72#diff-2e67cc4c3200608ae64f002575006135) from using `Console.ScrollBuffer`, `console.SetCursorPosition` and ` console.SetWindowPosition` to applying the escape sequence `"0x1b[2J"`.

That change was included in 2.0.0-beta4 and newer versions, this issue was reported to happen in 2.0.0-beta3.

@sorge13248 Please try out the 2.0.0-rc1 version of PSReadLine from PowerShell Gallery.

**Comment by daxian-dbw** [comment]
This issue had already been fixed by #790. The fix was included in 2.0.0-beta4 and newer versions. For more information, take a look at https://github.com/PowerShell/PSReadLine/issues/922#issuecomment-570475042.

Please try out the latest 2.0.0-rc1 version from PowerShell Gallery.
